go.sum

```diff
@@ -24,6 +24,7 @@ github.com/FactomProject/basen v0.0.0-20150613233007-fe3947df716e h1:ahyvB3q25Yn
 github.com/FactomProject/basen v0.0.0-20150613233007-fe3947df716e/go.mod h1:kGUqhHd//musdITWjFvNTHn90WG9bMLBEPQZ17Cmlpw=
 github.com/JohnCGriffin/overflow v0.0.0-20170615021017-4d914c927216 h1:2ZboyJ8vl75fGesnG9NpMTD2DyQI3FzMXy4x752rGF0=
 github.com/JohnCGriffin/overflow v0.0.0-20170615021017-4d914c927216/go.mod h1:X0CRv0ky0k6m906ixxpzmDRLvX58TFUKS2eePweuyxk=
+github.com/LaurentTrk/go-substrate-rpc-client v2.0.0-alpha.6.4+incompatible/go.mod h1:1LFrqbtH9YQgw/xKzQOlcnJEiHtiIL9N/67arAqO9fA=
```

Comment:
This seems like something we don't need to check in. Can you run `go mod tidy` on this?

Reply:
Sorry for that, golang newbie here ;)